geomesa-spark/geomesa-spark-accumulo/src/test/scala/org/locationtech/geomesa/spark/accumulo/SparkSQLTestUtils.scala

```diff
@@ -18,14 +18,26 @@ import org.geotools.data.simple.SimpleFeatureStore
 import org.geotools.data.{DataUtilities, DataStoreFinder}
 import org.geotools.geometry.jts.JTSFactoryFinder
 import org.joda.time.format.ISODateTimeFormat
+import org.locationtech.geomesa.accumulo.AccumuloProperties.AccumuloQueryProperties
 import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, AccumuloDataStoreParams}
 import org.locationtech.geomesa.features.ScalaSimpleFeature
+import org.locationtech.geomesa.index.conf.QueryProperties
 import org.locationtech.geomesa.tools.ingest.ConverterIngest
 import org.locationtech.geomesa.utils.geotools.{GeneralShapefileIngest, SimpleFeatureTypes}
 
 import scala.collection.JavaConversions._
 
 object SparkSQLTestUtils {
+  def setProperties(): Unit = {
+    System.setProperty(QueryProperties.SCAN_RANGES_TARGET.property, "1")
+    System.setProperty(AccumuloQueryProperties.SCAN_BATCH_RANGES.property, s"${Int.MaxValue}")
+  }
+
+  def clearProperties(): Unit = {
```

Comment:
thanks for updating this - now that i think about it some more, i still see 2 potential issues:

1. the tests might conflict with each other - if a test starts running while another one finishes, the properties might be cleared part-way through the second test. I believe tests are run concurrently in our builds, so this might be a problem
2. we set the SCAN_RANGES_TARGET property in our maven opts, so that our accumulo tests don't blow up (mock accumulo doesn't handle lots of ranges very well). So we need to cache the original value for the prop, and re-set it afterwards

There's actually a thread-local value for system properties that you can use instead of setting them globally - you can set and clear this in a 'beforeEachTest' and 'afterEachTest' method (I think in specs 2 you use the @before and @after annotations). You do something like:
```
AccumuloQueryProperties.SCAN_BATCH_RANGES.threadLocalValue.set("1")
...
AccumuloQueryProperties.SCAN_BATCH_RANGES.threadLocalValue.clear()
```